### PR TITLE
Feature/remove legacy auth

### PR DESCRIPTION
**Pull request by mr-nick17**
### What

Removed env config `ENABLE_NEW_SIGN_IN` 
Updated logic to handle removal of ⬆️ config
Removed legacy auth code
🤞 Fixed component tests so they run...

### How to review

Sense check
CI integration passes
Pull this branch, port forward to publishing api router on sandbox, login and check all new auth world still works ok

### Who can review

Frontend dev

### CODE REVIEW

**Review by cookel2 — CHANGES_REQUESTED** [review]
I haven't tested it yet but I've found a few typos.

File: src/app/utilities/http-methods/request.test.js [hunk 1]
```diff
@@ -14,6 +14,20 @@ jest.mock("../logging/log", () => {
         warn: jest.fn(() => {}),
     };
 });
+jest.mock("../../utilities/api-clients/user", () => {
+    class user {}
+    user.renewSession = () => {
+        return new Promise((resolve, reject) => {
```
[the remark this answers is not in the document]
Comment (mr-nick17) [reply src/app/utilities/http-methods/request.test.js:20]: Disagree, a promise resolves or rejects. You do not have to define what it does in the class definition as that's up to the calling function to handle it https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise

File: src/app/utilities/http-methods/request.test.js [hunk 2]
```diff
@@ -14,6 +14,20 @@ jest.mock("../logging/log", () => {
...
+            resolve({ expirationTime: mockSessionExpiryTime });
+        });
+    };
+    user.logOut = () => {
+        return new Promise((resolve, reject) => {
```
[the remark this answers is not in the document]
Comment (mr-nick17) [reply src/app/utilities/http-methods/request.test.js:25]: See similar comment

File: src/app/views/collections/details/CollectionDetailsController.jsx [hunk 3]
```diff
@@ -320,37 +320,35 @@ export class CollectionDetailsController extends Component {
...
-                        } else {
+        this.setState({ isFetchingUserDetails: true });
+        const updateEmailInActiveCollection = {
+            ...this.props.activeCollection,
+        };
+        const email = page.lastEditedBy ? page.lastEditedBy : page.lastEdit?.email;
+        await user
+            .getUser(email)
+            .then(response => {
+                const newActiveCollection = updateEmailInActiveCollection[state].map(collectionPage => {
```
[the remark this answers is not in the document]
Comment (mr-nick17) [reply src/app/views/collections/details/CollectionDetailsController.jsx:331]: Possibly but as these changes had poor test coverage I am reluctant to change the logic beyond removing the feature toggle

File: src/app/config/groups/thunks.js [hunk 4]
```diff
@@ -73,68 +73,23 @@ export const fetchGroupMembersRequest = id => dispatch => {
...
+    teams
+        .getGroups()
+        .then(response => {
+            dispatch(actions.loadGroupsSuccess(response.groups));
+        })
+        .catch(error => {
+            dispatch(actions.loadGroupsFailure());
+            notifications.add({
+                type: "warning",
+                isDismissable: true,
```
[the remark this answers is not in the document]
Comment (mr-nick17) [reply src/app/config/groups/thunks.js:87]: Whilst I agree that this is indeed a typo. My change is to remove the feature flag. To change this typo there would need to be changes across the app. I suggest this should be a bug fix for the future depending on priorities etc. especially as Florence is considered legacy

File: src/app/utilities/api-clients/user.js [hunk 5]
```diff
@@ -19,52 +19,44 @@ export default class user {
...
+                                autoDismiss: 5000,
+                            };
+                            notifications.add(notification);
+                            break;
+                        }
+                        default: {
+                            const notification = {
+                                type: "warning",
+                                message: errorCodes.GET_USERS_UNEXPECTED_ERROR_SHORT,
+                                isDismissable: true,
```
[the remark this answers is not in the document]
Comment (mr-nick17) [reply src/app/utilities/api-clients/user.js:55]: As above, invasive change

File: src/app/utilities/api-clients/user.js [hunk 6]
```diff
@@ -74,10 +66,15 @@ export default class user {
                     };
                     notifications.add(notification);
                 }
-                return error;
+            } else {
+                const notification = {
+                    type: "warning",
+                    message: errCodes.GET_USERS_UNEXPECTED_ERROR_SHORT,
+                    isDismissable: true,
```
[the remark this answers is not in the document]
Comment (mr-nick17) [reply src/app/utilities/api-clients/user.js:73]: As above, invasive change

**Review by cookel2 — APPROVED** [review]
It tested successfully - LGTM.

**Pull request merged**